elasticdl/python/worker/main.py

```diff
@@ -25,6 +25,46 @@
 CONNECT_PS_TIMEOUT = 300
 
 
+def build_ps_client(ps_addrs, logger):
```

Comment:
Should we move the function into ps_client.py?

Reply:
Add a docstring to describe the ps_addr.

Reply:
Good idea, updated.